### ISSUE TITLE
Background forced to white instead of transparent in Post loop builder

### ISSUE THREAD

**Issue by brandforwardnl** [soflyy/breakdance-bugs#864]
**What version of Breakdance are you using?**

For example: v1.3.1

**What browser are you using?**

Safari (Latest)

**What operating system are you using?**

MacOS (Latest)

**Reproduction  Steps**

I want to use the post loop builder in a section with a background. It appears that the background color is always white if you don't select a color where it should be transparent. I also tried this:
- a transparent pixel.png --> Results in a white BG
- a gradient where both colors are set to transparent --> Results in white BG
- Added custom CSS to make the BG transparent --> Results in white BG

To reproduce:
- Create a global block with something in it and don't select a background color (or a fully transparent background)
- Create a new page with a section. 
- Give a nice background image to this section
- Add the post loop elemento to this page and select the global block that you created

As you can see the background of the loop element is white where you'd expect to see the background of the section behind it.

**Reproduction Video**

Here you can see how it looks in the video:
https://share.cleanshot.com/CVHZCq8r

No matter what I try: the background won't get transparent.

**Comment by a-dubiel** [comment]
Did you try setting background on the global block element?

**Comment by brandforwardnl** [comment]
Yes, I tried everything. Global block post background, container background, placing the global block in a div and making the background of that div transparent, tried with custom CSS: background-color: #FFFFFF00 and background-color:transparent; but it just won't be transparent.

**Comment by brandforwardnl** [comment]
Hi @a-dubiel,

I tried that (no image, color or gradient) but that gives the same result: white BG instead of transparent.

**Comment by StaggerLeee** [comment]
I have it for one section and have no clue where it is set. On frontend it is OK, just backend when editing Page template white background wont go away. I can set new color but not remove this white-gray one. Opacitiy cannot make it go away.

Deleted browser cache and cokies.

```
.breakdance {
    background-color: var(--bde-background-color);
    color: var(--bde-body-text-color);
}
```

```
:root {
...
--bde-background-color: var(--grey-50);
...
}
```

**Comment by StaggerLeee** [comment]
I solved it by setting color in Global settings - Colors - Background to some color and full opacity. Have to come back to this issue to remember to remove it when editing is finished.

**Comment by brandforwardnl** [comment]
Nice ;-)

Met vriendelijke groet,

Rob de Wit

Mobiel: 06 2113 09 14

Telefoon: 030 227 1240

E-mail: ***@***.***

Web: [Brandforward.nl](https://brandforward.nl)

Lange Nieuwstraat 66, 3512PL Utrecht

[Tevreden over onze dienstverlening? Schrijf een review!](https://g.page/r/CXAKBI2EGJclEAQ/review)	https://g.page/r/CXAKBI2EGJclEAQ/review

On July 13, 2023 at 12:44 GMT, Andrzej Dubiel ***@***.***> wrote:

Closed [#864](https://github.com/soflyy/breakdance-bugs/issues/864) as completed.
